pkg/rpm/unit.spec.in

```diff
@@ -193,20 +194,19 @@ fi
 %defattr(-,root,root,-)
 %attr(0755,root,root) %{_sbindir}/unitd
 %attr(0755,root,root) %{_sbindir}/unitd-debug
-%if ! %{use_systemd}
-%config(noreplace) %{_sysconfdir}/sysconfig/unit
-%endif
 %dir %{_sysconfdir}/unit
 %if %{use_systemd}
 %{_unitdir}/unit.service
 %else
+%config(noreplace) %{_sysconfdir}/sysconfig/unit
```

Comment:
@remicollet it seems like this one is mistakenly reverted here from 
https://github.com/nginx/unit/pull/212/commits/575d6d42146a6f0138d0adf9dfb7b90422d9e52f ?

Reply:
Nop, simply move lower (to avoid multiple conditions)